test/fixtures/hpa/hpa/deployment.yml

```diff
@@ -0,0 +1,24 @@
+apiVersion: apps/v1beta1
```

Comment:
Why is this fixture set two layers deep? i.e. why not `test/fixtures/hpa/thing.yml`?